kraken-std/src/kraken/std/cargo/__init__.py

```diff
@@ -357,6 +363,7 @@ def cargo_test(
     group: str | None = "test",
     project: Project | None = None,
     features: list[str] | None = None,
+    depends_on: Sequence[Task] | None = None,
```

Comment:
```suggestion
    depends_on: Sequence[Task] = (),
```

Then you don't need the `if` below.